pennylane/gradients/gradient_transform.py

```diff
@@ -87,7 +87,7 @@ class gradient_transform(qml.batch_transform):
 
     .. code-block:: python
 
-        @quantum_gradient
+        @gradient_transform

```

Comment:
I cannot seem to leave a suggestion, but on line 59 above, change outputs -> output.

Reply:
done!